src/components/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
-/* eslint-disable no-unused-vars */
 import calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <>
-      <Display result="" />
-      <ButtonPanel />
-    </>
-  );
+class App extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(buttonName) {
+    console.log(buttonName);
+    const { total, next, operation } = calculate(this.state, buttonName);
+    this.setState({
+      total,
+      next,
+      operation,
+    });
+    console.log(total, next, operation);
+  }
+
+  // total null,
```

Comment:
- [ ] It will be great to delete commented code in your application. This ensures that your code is clean. Remember you can always use git history to go back to the previous version of your code ⬇️ 
<img width="792" alt="no_inline_comments" src="https://user-images.githubusercontent.com/32361963/129004447-60b5b508-159d-4029-8c61-57e6dfa7dd59.png">